packages/flutter/test/widgets/custom_painter_test.dart

```diff
@@ -356,6 +356,7 @@ void _defineTests() {
     final Set<SemanticsAction> allActions = SemanticsAction.values.toSet()
       ..remove(SemanticsAction.customAction) // customAction is not user-exposed.
       ..remove(SemanticsAction.showOnScreen) // showOnScreen is not user-exposed
+      // TODO(LongCatIsLooong): https://github.com/flutter/flutter/issues/159515.
```

Comment:
What do these TODO's refer to?  I believe the format should be:

```c++
// TODO(159515): This is the thing we should do.
```

Reply:
Per the style guide: https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#avoid-checking-in-comments-that-ask-questions

I'll add "clean up after `SemanticsAction.scrollToOffset` is added in dart:ui" to the description.

Reply:
As someone that is not you, I'm not sure what cleaning up entails.  There isn't a clear action that someone other than you could take.  A verb phase like "remove this" is more descriptive than "clean up".